ksqldb-engine/src/test/java/io/confluent/ksql/engine/KsqlEngineTest.java

```diff
@@ -1445,7 +1445,7 @@ public void shouldCleanUpInternalTopicsOnQueryCloseForPersistentQueriesSharedRun
   }
 
   @Test
-  public void shouldNotHardDeleteSubjectForPersistentQuery() throws IOException, RestClientException {
+  public void shouldHardDeleteSubjectForPersistentQuery() throws IOException, RestClientException {
```

Comment:
we should have a test that for persistent queries we only clean up _internal_ subjects and not any input or output topic subjects.